<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumeWidget</class>
 <widget class="QWidget" name="VolumeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Volume</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/volume_black.svg</normaloff>../icons/volume_black.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSlider" name="volume_slider">
       <property name="styleSheet">
        <string notr="true">selection-color: rgb(255, 255, 255);
gridline-color: rgb(255, 255, 255);
selection-background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="sliderPosition">
        <number>99</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
